web/viewer.css

```diff
@@ -487,14 +489,52 @@ select {
   background-color: rgba(255, 102, 102, 1);
 }
 
-.secondaryToolbar {
+.secondaryToolbar,
+.editorParamsToolbar {
   padding: 6px 0 10px;
   inset-inline-end: 4px;
   height: auto;
   z-index: 30000;
   background-color: var(--doorhanger-bg-color);
 }
 
+#editorParamsToolbarContainer > .editorParamsSetter {
+  max-width: 220px;
+  min-height: 26px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 12px;
+  padding-right: 12px;
+}
+
+#editorParamsToolbarContainer .editorParamsLabel {
+  padding-left: 10px;
+  flex: none;
+}
+
+#editorParamsToolbarContainer .editorParamsColor {
+  width: 32px;
+  height: 32px;
+  flex: none;
+}
+
+#editorParamsToolbarContainer .editorParamsSlider {
+  background-color: transparent;
+  width: 90px;
+  flex: 0 1 0;
+}
+
+#editorParamsToolbarContainer .editorParamsSlider::-moz-range-progress {
+  background-color: black;
+}
+#editorParamsToolbarContainer .editorParamsSlider::-moz-range-track {
+  background-color: black;
+}
+#editorParamsToolbarContainer .editorParamsSlider::-moz-range-thumb {
+  background-color: white;
+}
```

Comment:
Are these not available as standard CSS features instead (since it won't work in the GENERIC viewer)?